src/java/org/apache/cassandra/schema/SchemaConstants.java

```diff
@@ -72,9 +74,47 @@ public final class SchemaConstants
 
     public static final List<String> LEGACY_AUTH_TABLES = Arrays.asList("credentials", "users", "permissions");
 
+    /**
+     * Checks if the length of the given name will be suitable to be used
+     * in constructed file names.
```

Comment:
Maybe clarify that by "file names" you mean e.g. "names of sstable files"

Reply:
I've seen usage in different file types: in addition to sstable files, log files, cache files, index files.